pinot-query-planner/src/main/java/org/apache/pinot/query/planner/physical/DispatchablePlanVisitor.java

```diff
@@ -82,6 +83,13 @@ public Void visitWindow(WindowNode node, DispatchablePlanContext context) {
     return null;
   }
 
+  @Override
+  public Void visitSetOp(SetOpNode setOpNode, DispatchablePlanContext context) {
```

Comment:
changing the dispatchable plan visitor requires a Ser/De test added to the QueryDispatcherTest and QueryServerTest to ensure all stagenode related content are ser/de properly

Reply:
added.